src/rest/SearchEndpoint.ts

```diff
@@ -814,6 +816,37 @@ export class SearchEndpoint implements ISearchEndpoint {
     return this.getQuerySuggest(request, callOptions, callParams);
   }
 
+  /**
+   * Searches through the values of a facet.
+   * @param request The request for which to search through the values of a facet.
+   * @param callOptions An additional set of options to use for this call.
+   * @param callParams The options injected by the applied decorators.
+   * @returns {Promise<IFacetSearchResponse>} A Promise of facet search results.
+   */
+  @path('/facet')
+  @method('POST')
+  @requestDataType('application/json')
+  @responseType('text')
```

Comment:
I guess we would also want to send :

```
@includeActionsHistory()
@includeReferrer()
@includeVisitorId()
@includeIsGuestUser()
```

These are used by the ML, to do some personalization on search results.